plugins/kubernetes/app/views/samson_kubernetes/_deploy_changeset_tab_nav.html.erb

```diff
@@ -1,3 +1,3 @@
-<% if @deploy.kubernetes %>
+<% if @deploy ? @deploy.kubernetes : @stage&.kubernetes? %>
```

Comment:
revert ?

Reply:
still needed (although updated to @deploy.persisted) since the new deploy doesn't have kubernetes set.